client/app/components/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import AntdButton, { ButtonProps as AntdButtonProps } from "antd/lib/button";
+import classNames from "classnames";
+import React from "react";
+
+import "./Button.less";
+
+interface ButtonProps extends Omit<AntdButtonProps, "type"> {
+  type?: AntdButtonProps["type"] | "plain";
+}
+
+function Button({ type, className, ...props }: ButtonProps) {
+  return type === "plain" ? (
+    <button className={classNames("btn-plain", "clickable", className)} type="button" {...props} />
+  ) : (
+    <AntdButton type={type} {...props} />
+  );
+}
```

Comment:
That's a good step forward 🎉  Let's improve it a bit more: `Button` is quite a generic name for this component, and it's a bit abused this way. Also, it creates some confusion with Ant's Button - it's hard to tell which should be used in particular place. Let's just choose less generic name for this component (`PlainButton`, `ButtonLink` or sth like that) and make it render just simply a button that looks like a link. Also, rename `btn-plain` class after component's name (currently `btn-plain` looks like a part of Bootstrap)